src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -127,6 +128,13 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
             }
             commit(ModelCalibrateMutation.SetComparisonPlotData, response.data);
         }
+    },
+
+    async resumeCalibrate(context) {
+        const {dispatch, state} = context;
+        if (state.calibrating && !state.complete) {
+            await dispatch("poll");
+        }
     }
```

Comment:
An alternative would be to do the logic check for `state.calibrating && !state.complete` in the component and dispatch `poll` from there to save making another action. I think we've tended to put quite a lot of logic like that into the components, though you could argue it belongs more in the store. Either way is ok I think.

Reply:
I'm happy to move this into the component if you think that is more standard, but if you're not feeling too strong I'm tempted to leave as is out of laziness